README.md

```diff
@@ -47,6 +47,10 @@ You do not have to create a dedicated token. Make sure to use the GitHub's defau
 **Optional** By default ZAP Docker container will fail with an [exit code](https://github.com/zaproxy/zaproxy/blob/efb404d38280dc9ecf8f88c9b0c658385861bdcf/docker/zap-baseline.py#L31), 
 if it identifies any alerts. Set this option to `true` if you want to fail the status of the GitHub Scan if ZAP identifies any alerts during the scan.  
 
+### `allow_issue_writing`
+
+**Optional** If set to true, the action will file the report to the GitHub issue using the issue_title input. If it is false, no issue will be created or updated.
```

Comment:
Maybe we should mention that by default it will create/update an issue, if no inputs have been provided.

**Optional** By default the baseline action will file the report to the GitHub issue using the issue_title input.
Set this to false if you don't want the issue to be created or updated.